### DISCUSSION TITLE
Find Books With Multiple Files For Audiobook

### DISCUSSION THREAD

**Discussion by jrpetersjr** [advplyr/audiobookshelf#2389] [General]
Hi,

Is there a way that in ABS I can see just the books that have multiple files making up the audiobook? I'd like to go through them all and use ABS to convert them all to one single M4B, but I don't want to have to click through every single audiobook that I have. I only have about 100 currently, but that number will be rising quickly. I know that I can just go through each book, but I'm looking for a faster way. 

TIA

**Comment by nichwall** [comment]
Yes, you can use the Filter at the top. Filter -> Tracks -> Multi track

**Comment by jrpetersjr** [comment]
@nichwall - Thanks for the info! It's been very useful!
